### PR TITLE
Fix live snapshots

### PR DESCRIPTION
**Pull request by alop**
The live snapshots method uses 'extract_snapshots' which we had
an older version of. The latest H4 tarball shows that we do not
pass the name, but the dirty merge we did for the rbd clone did.
As such, when we were doing live snaps, we'd raise the error
TypeError: extract_snapshot() takes exactly 5 arguments (4 given)
as it was expecting the name to be passed.

Not-in-upstream: False
Closes-rally-bug: DE1509
Upstream-Review: https://review.openstack.org/#/c/78529/

### CODE REVIEW

File: nova/virt/libvirt/utils.py [hunk 1]
```diff
@@ -576,7 +576,7 @@ def delete_snapshot(disk_path, snapshot_name):
     execute(*qemu_img_cmd, run_as_root=True)
 
 
-def extract_snapshot(disk_path, source_fmt, snapshot_name, out_path, dest_fmt):
+def extract_snapshot(disk_path, source_fmt, out_path, dest_fmt):
     """Extract a named snapshot from a disk image
```
Comment (signed8bit) [inline nova/virt/libvirt/utils.py]: Can we update the comments to omit mention of snapshot_name as per upstream?

```
"""Extract a snapshot from a disk image.
Note that nobody should write to the disk image during this operation.
:param disk_path: Path to disk image
:param out_path: Desired path of extracted snapshot
"""
```

**Comment by signed8bit** [comment]
@alop Can we also dig up the upstream review so we can tag the backport?

**Comment by alop** [comment]
@signed8bit https://review.openstack.org/#/c/78529/

File: nova/virt/libvirt/utils.py (hunk 1 above)
Comment (alop) [inline nova/virt/libvirt/utils.py]: done

**Comment by signed8bit** [comment]
Excellent, can you amend the commit to add  the Upstream-Review: https://review.openstack.org/#/c/78529/ tag?

**Comment by alop** [comment]
Done

**Comment by signed8bit** [comment]
+1

**Pull request merged**